docs/manifest-types.md

```diff
@@ -110,6 +110,49 @@ This manifest adds a constant to specified target class. To set value you can sp
 }
 ```
 
+This manifest set expression to specified method of target class. To set value you can specify value field. It can be everything supported by PHP.
+
+```json
+{
+    "configure-module": [
+        {
+            "target": "\\Spryker\\Client\\Catalog\\CatalogConfig::targetMethod",
+            "value": 10
+        }
+    ]
+}
+```
+
+This manifest adds expression to specified method of target class. To set value you can specify value field. It can be everything supported by PHP.
+
+```json
+{
+    "configure-module": [
+        {
+            "target": "\\Spryker\\Client\\Catalog\\CatalogConfig::targetMethod",
+            "value": [
+                10
+            ]
+        }
+    ]
+}
+```
+
+This manifest substitutes expression of specified method target class in case if not exists any changes in according target method, checked by `previousValue` parameter. To set value you can specify value field. It can be everything supported by PHP.
```

Comment:
"This manifest substitutes expression of specified method target class in case if not exists any changes in according target method, checked by `previousValue` parameter."

TBH if I have no context - this has no sense

Reply:
Changed to:
This manifest substitutes expression of specified method of target class in case if statements of target class method the same as in `previousValue` parameter. To set value you can specify value field. It can be everything supported by PHP.